src/main/java/com/github/teamfusion/spyglassplus/client/render/item/SpyglassStandBWLR.java

```diff
@@ -0,0 +1,37 @@
+package com.github.teamfusion.spyglassplus.client.render.item;
+
+import com.github.teamfusion.spyglassplus.SpyglassPlus;
+import com.github.teamfusion.spyglassplus.client.ModModelLayer;
+import com.github.teamfusion.spyglassplus.client.model.SpyglassStandModel;
+import com.github.teamfusion.spyglassplus.core.registry.SpyglassPlusItems;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.blaze3d.vertex.VertexConsumer;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.BlockEntityWithoutLevelRenderer;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.block.model.ItemTransforms;
+import net.minecraft.client.renderer.entity.ItemRenderer;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+
+public class SpyglassStandBWLR extends BlockEntityWithoutLevelRenderer {
+	private static final ResourceLocation SPYGLASS_STAND_LOCATION = new ResourceLocation(SpyglassPlus.MOD_ID, "textures/entity/spyglass_stand.png");
+
+	private SpyglassStandModel model;
+
+	public SpyglassStandBWLR() {
+		super(Minecraft.getInstance().getBlockEntityRenderDispatcher(), Minecraft.getInstance().getEntityModels());
```

Comment:
This is unsafe. The class isn't only in the Client and the Minecraft class is client only. If this class should be Client only, annotate it as such. Otherwise, you'll need to find a different way to do this.